src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mlp_seq.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/mlp.hpp"
+
+#include "utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+namespace {
+
+std::vector<std::vector<InputShape>> inputShape_2D(bool with_dynamic = true) {
+    auto shapes = SNIPPETS_TESTS_STATIC_SHAPES(
+        {{1, 64}},
+        {{2, 64}},
+        {{4, 64}},
+        {{8, 64}},
+        {{16, 64}},
+        {{32, 64}},
+        {{64, 64}});
+    if (with_dynamic) {
+        shapes.push_back({{PartialShape{-1, 64}, {{1, 64}, {8, 64}, {8, 64}, {6, 64}}}});
+        shapes.push_back({{PartialShape{-1, 64}, {{2, 64}, {2, 64}, {4, 64}, {3, 64}}}});
+    }
+    return shapes;
+}
+
+std::vector<size_t> numHiddenLayers() {
+    return {1, 2, 3, 5, 7, 9};
+}
+std::vector<size_t> hiddenMatmulSizes() {
+    return {16, 32, 64, 128};
+}
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MLP_SEQ_2D_f32,
+                         MLP,
+                         ::testing::Combine(::testing::ValuesIn(inputShape_2D()),
+                                            ::testing::ValuesIn(precision_f32(1)),
+                                            ::testing::Values(ov::element::f32),
+                                            ::testing::ValuesIn({static_cast<size_t>(1), MLP::default_thread_count}),
+                                            ::testing::Values(1),  // Subgraph
+                                            ::testing::Values(1),  // MLP
+                                            ::testing::Values(ov::test::utils::DEVICE_CPU),
+                                            ::testing::Values(CPUTestUtils::empty_plugin_config),
+                                            ::testing::ValuesIn(numHiddenLayers()),
+                                            ::testing::ValuesIn(hiddenMatmulSizes())),
+                         MLP::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MLP_SEQ_Quantized_2D_f32,
```

Comment:
Could you please check if we can combine all `MLPQuantized` instances into one? At the first glance, it seems like we can just parametrize 2nd (i8, f32) and 3rd (f32, bf16) parameters